nestkernel/simulation_manager.cpp

```diff
@@ -375,6 +376,61 @@ nest::SimulationManager::simulate( Time const& t )
   t_slice_begin_ = timeval();
   t_slice_end_ = timeval();
 
+  // Reset profiling timers and counters within event_delivery_manager
+  kernel().event_delivery_manager.reset_timers_counters();
```

Comment:
Is the nestio backend being re-initialized with each `run` in this set up, after we merge? And do we want to have different measurements for each run? It's a non-rhetorical question, what should the semantics here be. We want as little as possible being reinitialized in run, so that you can use run to handle interrupts or do external updates down from the python layer at regular intervals.